pages/dashboard/team-page.js

```diff
@@ -153,7 +153,7 @@ exports.TeamPage = class TeamPage extends BasePage {
   }
 
   async isTeamDeleted(teamName) {
-    await this.page.waitForTimeout(5000)
+    await this.page.getByText('Your Penpot').waitFor({ state: 'visible' , timeout: 8000});
```

Comment:
I'd also use the `toBeVisible` assertion:
`await expect(this.page.getByText('Your Penpot')).toBeVisible({timeout: 8000});`

Reply:
done